go-controller/pkg/util/multi_network.go

```diff
@@ -939,6 +939,13 @@ func GetPodNADToNetworkMappingWithActiveNetwork(pod *kapi.Pod, nInfo NetInfo, ac
 		Name:      activeNetworkNADKey[1],
 	}
 
+	if nInfo.IsPrimaryNetwork() && nInfo.TopologyType() == types.Layer2Topology && nInfo.AllowsPersistentIPs() {
```

Comment:
it would probably be useful to have a free `AllowsPersistentIPs(netInfo)` method here that summarizes which network allow persistent IPs

Reply:
OK, I'll add it.

Reply:
Done.